development/input-app/pages/input/index.mdx

```diff
@@ -9,6 +9,7 @@ import DisabledInput from "../../components/input/DisabledInput/DisabledInput.ts
 import PasswordInput from "../../components/input/PasswordInput/PasswordInput.tsx";
 import PasswordShowHide from "../../components/input/PasswordShowHide/PasswordShowHide.tsx";
 import NumberInput from "../../components/input/NumberInput/NumberInput.tsx";
+import Forwardrefinput from "../../components/input/Forwardrefinput/Forwardrefinput.jsx"
```

Comment:
Import and use tsx file here. jsx file is for viewing purpose only.

import Forwardrefinput from "../../components/input/ForwardRefInput/ForwardRefInput.tsx"